src/Controller/ChangePasswordController.php

```diff
@@ -42,6 +43,8 @@ public function changePasswordAction(Request $request, UserPasswordHasherInterfa
             $this->getEM()->persist($user);
             $this->getEM()->flush();
 
+            $userNotifier->notifyChange($user->getEmail(), 'Your password has been changed');
```

Comment:
It would be nice to reuse this UserNotifier for the 2fa emails too I think, to keep it all the same, if you don't mind refactoring that. It probably needs the ability to pass in a custom email template filename tho as the 2fa emails contain some extra wording.

Reply:
I don't see this done but no worries I'll do it after merging :)

Reply:
Woops I havent add the file sorry x)
I can made a PR fixing that later if you want :)